browser(firefox-beta): roll Firefox beta to 96.0b5

aslushnikov opened the pull request:
Review URL: https://github.com/aslushnikov/juggler/commit/4cedc704b15ac9926819d79b04407d4f9a09108e

Firefox 96 changes default sameSite cookie value, so this patch
updates tests to accomodate for this.

yury-s commented on tests/browsercontext-route.spec.ts:
```diff
@@ -123,8 +123,9 @@ it('should support Set-Cookie header', async ({ contextFactory, server, browserN
     });
   });
   await page.goto('https://example.com');
+  const defaultSameSiteCookieValue = browserName === 'chromium' || (browserName === 'firefox' && browserMajorVersion >= 96) ? 'Lax' : 'None';
```
Revert changes in the tests

aslushnikov replied:
discussed offline and decided to land here

aslushnikov replied:
reverted

aslushnikov committed:
browser(firefox-beta): roll Firefox beta to 96.0b5

Review URL: https://github.com/aslushnikov/juggler/commit/4cedc704b15ac9926819d79b04407d4f9a09108e

Firefox 96 changes default sameSite cookie value, so this patch
updates tests to accomodate for this.
